[IMP] sales: update quote_template for 16 (#5873)
odoo/documentation: content/applications/sales/sales/send_quotations/quote_template.rst

===================

In Odoo *Sales*, salespeople have the ability to create reusable quotation templates for common
products or services that the business offers.

By using these templates, quotations can be tailored and sent to customers at a much faster pace,
without having to create new quotations from scratch every time a sales negotiation occurs.

Configuration
=============

Begin by activating the setting in :menuselection:`Sales app --> Configuration --> Settings`, and

> scroll to the :guilabel:`Quotations & Orders` heading.

Reviewer: I totally understand your suggestion here, but I think it's actually alright in its current state. It's intended to be read as "Go to X,Y,Z -- and scroll to A,B,C" -- does that make sense. However, if you think it's really in need of a change, just let me know, and I'll go back and make the necessary change. Thanks!

> scroll to the :guilabel:`Quotations & Orders` heading.

Reviewer 2: This was more so a preference thing as I was reading it, but it works just fine the way you have it so you're good to keep it that way!

:guilabel:`Add a note` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in which
the desired note can be typed. When the note has been entered, click away to secure the note.

Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon, located
to the left of each line item.

To delete any line item from the :guilabel:`Lines` tab (product, section, and/or note), click the
:guilabel:`🗑️ (trash can)` icon on the far-right side of the line.

Optional Products tab
---------------------

The use of optional products is a marketing strategy that involves the cross-selling of products
along with a core product. The aim is to offer useful and related products to customers, which may
result in an increased sale.

For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
products enhances the customer experience.

Optional products appear as a section on the bottom of sales orders and eCommerce pages. Customers
can immediately add them to their online sales orders themselves, if desired.

> In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each compatible cross-selling

Reviewer 3: "Compatible" might sound like some products are not eligible to be optional products if they're not compatible
```suggestion
In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling
```

product related to the original items in the :guilabel:`Lines` tab, if applicable. The products
added here ideally compliment the original offering as added value for the prospective buyer.

.. image:: quote_template/optional-products-tab-quotation-template.png
   :align: center
   :alt: Populated optional products tab on a quotation template in Odoo Sales.

> To add an optional product to a quotation template, click :guilabel:`Add a line` in the
> :guilabel:`Optional Products` tab. Doing so reveals a blank field in the :guilabel:`Product` column.

Reviewer 3: This paragraph is a bit redundant with the previous one---shortening it would help with that
```suggestion
Clicking :guilabel:`Add a line` reveals a blank field in the :guilabel:`Product` column.
```

Terms & Conditions tab
----------------------

The :guilabel:`Terms & Conditions` tab provides the opportunity to add terms and conditions to the
quotation template. To add terms and conditions to the quotation template, simply type (or
copy/paste) the desired terms and conditions in this tab.

.. image:: quote_template/terms-and-conditions-tab.png
   :align: center
   :alt: Terms and conditions tab in a quotation template form in Odoo Sales.

.. seealso::
   - :doc:`/applications/finance/accounting/customer_invoices/terms_conditions`

.. note::
   Terms and conditions are **not** required.

Design quotation templates
==========================

In the upper-left corner of the quotation template form, there's a :guilabel:`Design Template`
button.

.. image:: quote_template/design-template-button.png
   :align: center
   :alt: Design template button in the upper-left corner of quotation template form.

When clicked, Odoo reveals a preview of the quotation template, through the Odoo *Website*
application, as it will appear on the front-end of the website to the customer.

Odoo uses numerous blue placeholder blocks to signify where certain elements appear, and what they

> contain (e.g. :guilabel:`Template Header`, :guilabel:`Product`, etc.).

Reviewer 3: ```suggestion
contain (e.g. :guilabel:`Template Header`, :guilabel:`Product`).
```

any unique need, look, or style.

.. tip::
   Quotation template design uses the same methodology and functionality with design building blocks
   as a typical web page design with Odoo *Website*. Be sure to check out the
   :doc:`/applications/websites/website` documentation to learn more.

When all blocks and customizations are complete, click the :guilabel:`Save` button to put those
configurations into place.

There is also a blue banner at the top of the quotation template design with a link to quickly
return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation template form in
the back-end of the *Sales* application.

Use quotation templates
=======================

When creating a quotation (:menuselection:`Sales app --> New`), choose a pre-configured template in
the :guilabel:`Quotation Template` field drop-down menu.

.. image:: quote_template/quotation-templates-field.png
   :align: center
   :alt: Quotation templates field on a standard quotation form in Odoo Sales.

To view what the customer will see, click the :guilabel:`Preview` button at the top of the page to

> see how the quotation template appears on the front-end of the website.

Reviewer 3: ```suggestion
see how the quotation template appears on the front-end of the website through Odoo's customer portal.
```

:guilabel:`🗑️ (trash can)` icon on the far-right side of the line.

Optional Products tab
---------------------

The use of *optional products* is a marketing strategy that involves the cross-selling of products
along with a core product. The aim is to offer useful and related products to customers, which may
result in an increased sale.

For instance, if a customer wants to buy a car, they have the choice to order massaging seats, as
well, or ignore the offer and simply buy the car. Presenting the choice to purchase optional
products enhances the customer experience.

Optional products appear as a section on the bottom of sales orders and eCommerce pages. Customers
can immediately add them to their online sales orders themselves, if desired.

.. image:: quote_template/optional-products-on-sales-order.png
   :align: center
   :alt: Optional products appearing on a typical sales order with Odoo Sales.

In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling product
related to the original items in the :guilabel:`Lines` tab, if applicable. The products added here

> ideally compliment the original offering as added value for the prospective buyer.

Reviewer 3: ```suggestion
ideally complement the original offering as added value for the prospective buyer.
```

.. image:: quote_template/optional-products-tab-quotation-template.png
   :align: center
   :alt: Populated optional products tab on a quotation template in Odoo Sales.

Clicking :guilabel:`Add a line` reveals a blank field in the :guilabel:`Product` column.

When clicked, a drop-down menu with products from the database appear. Select the desired product
from the drop-down menu to add it as an optional product to the quotation template.

To delete any line item from the :guilabel:`Optional Products` tab, click the :guilabel:`🗑️ (trash
can)` icon.

.. note::
   Optional products are **not** required.

Terms & Conditions tab
----------------------

The :guilabel:`Terms & Conditions` tab provides the opportunity to add terms and conditions to the

> quotation template. To add terms and conditions to the quotation template, simply type (or

Reviewer 3: Just a bit redundant from the previous sentence
```suggestion
quotation template. To add terms and conditions, simply type (or
```

copy/paste) the desired terms and conditions in this tab.

.. image:: quote_template/terms-and-conditions-tab.png
   :align: center
   :alt: Terms and conditions tab in a quotation template form in Odoo Sales.

.. seealso::
   - :doc:`/applications/finance/accounting/customer_invoices/terms_conditions`

.. note::
   Terms and conditions are **not** required.

Design quotation templates
==========================

In the upper-left corner of the quotation template form, there's a :guilabel:`Design Template`
button.

.. image:: quote_template/design-template-button.png
   :align: center
   :alt: Design template button in the upper-left corner of quotation template form.

When clicked, Odoo reveals a preview of the quotation template, through the Odoo *Website*

> application, as it will appear on the front-end of the website to the customer.

Reviewer 3: We may want to state explicitly somewhere in this section (maybe as a note?) that this feature is only available if you have the Website app activated

and feature-rich building blocks. These building blocks can be dragged-and-dropped anywhere on the
quotation template design.

.. image:: quote_template/design-quotation-building-blocks.png
   :align: center
   :alt: Design quotation template building blocks sidebar in Odoo Website.

After a block has been dropped in the desired position, it can be customized and configured to fit
any unique need, look, or style.

.. tip::
   Quotation template design uses the same methodology and functionality with design building blocks
   as a typical web page design with Odoo *Website*. Be sure to check out the
   :doc:`/applications/websites/website` documentation to learn more.

When all blocks and customizations are complete, click the :guilabel:`Save` button to put those
configurations into place.

There is also a blue banner at the top of the quotation template design with a link to quickly
return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation template form in
the back-end of the *Sales* application.

Use quotation templates
=======================

When creating a quotation (:menuselection:`Sales app --> New`), choose a pre-configured template in

> the :guilabel:`Quotation Template` field drop-down menu.

Reviewer 3: ```suggestion
the :guilabel:`Quotation Template` field.
```

   :guilabel:`Product` field, and the option appears in the drop-down menu. Products can also be
   found by clicking :guilabel:`Search More...` from the drop-down menu.

.. note::
   When a product is added to a quotation template, the default :guilabel:`Quantity` is `1`, but
   that can be edited at any time.

Then, drag-and-drop the product to the desired position, via the :guilabel:`six squares` icon,
located to the left of each line item.

To add a *section*, which serves as a header to organize the lines of a sales order, click
:guilabel:`Add a section` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in
which the desired name of the section can be typed. When the name has been entered, click away to
secure the section name.

Then, drag-and-drop the section name to the desired position, via the :guilabel:`six squares` icon,
located to the left of each line item.

To add a note, which would appear as a piece of text for the customer on the quotation, click
:guilabel:`Add a note` in the :guilabel:`Lines` tab. When clicked, a blank field appears, in which
the desired note can be typed. When the note has been entered, click away to secure the note.

> Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon, located
> to the left of each line item.

Reviewer 2: ```suggestion
Then, drag-and-drop the note to the desired position, via the :guilabel:`six squares` icon.
```
I feel like we've been told where it is a few times now, so to avoid redundancy and keep it short/sweet, I deleted it here. But, it's just a suggestion so ignore if you'd like to keep it

quotation template. To add terms and conditions to the quotation template, simply type (or
copy/paste) the desired terms and conditions in this tab.

.. image:: quote_template/terms-and-conditions-tab.png
   :align: center
   :alt: Terms and conditions tab in a quotation template form in Odoo Sales.

.. seealso::
   - :doc:`/applications/finance/accounting/customer_invoices/terms_conditions`

.. note::
   Terms and conditions are **not** required.

Design quotation templates
==========================

In the upper-left corner of the quotation template form, there's a :guilabel:`Design Template`
button.

.. image:: quote_template/design-template-button.png
   :align: center
   :alt: Design template button in the upper-left corner of quotation template form.

When clicked, Odoo reveals a preview of the quotation template, through the Odoo *Website*
application, as it will appear on the front-end of the website to the customer.

Odoo uses numerous blue placeholder blocks to signify where certain elements appear, and what they
contain (e.g. :guilabel:`Template Header`, :guilabel:`Product`).

> To edit the content, look, and overall design of the quotation template, via the *Website*

Reviewer 2: ```suggestion
To edit the content, appearance, and overall design of the quotation template via the *Website*
```

Optional products appear as a section on the bottom of sales orders and eCommerce pages. Customers
can immediately add them to their online sales orders themselves, if desired.

.. image:: quote_template/optional-products-on-sales-order.png
   :align: center
   :alt: Optional products appearing on a typical sales order with Odoo Sales.

In the :guilabel:`Optional Products` tab, :guilabel:`Add a line` for each cross-selling product
related to the original items in the :guilabel:`Lines` tab, if applicable. The products added here
ideally complement the original offering as added value for the prospective buyer.

.. image:: quote_template/optional-products-tab-quotation-template.png
   :align: center
   :alt: Populated optional products tab on a quotation template in Odoo Sales.

Clicking :guilabel:`Add a line` reveals a blank field in the :guilabel:`Product` column.

When clicked, a drop-down menu with products from the database appear. Select the desired product
from the drop-down menu to add it as an optional product to the quotation template.

To delete any line item from the :guilabel:`Optional Products` tab, click the :guilabel:`🗑️ (trash
can)` icon.

.. note::

>    Optional products are **not** required.

Reviewer 4: Optional, but I think being specific here is helpful 🙂
```suggestion
   Optional products are **not** required to create a quotation template.
```

Terms & Conditions tab
----------------------

The :guilabel:`Terms & Conditions` tab provides the opportunity to add terms and conditions to the
quotation template. To add terms and conditions, simply type (or copy/paste) the desired terms and
conditions in this tab.

.. image:: quote_template/terms-and-conditions-tab.png
   :align: center
   :alt: Terms and conditions tab in a quotation template form in Odoo Sales.

.. seealso::

>    - :doc:`/applications/finance/accounting/customer_invoices/terms_conditions`

Reviewer 4: Can remove single bullet point from seealso, and update doc path to be relative
```suggestion
   :doc:`../../../finance/accounting/customer_invoices/terms_conditions`
```

.. note::

>    Terms and conditions are **not** required.

Reviewer 4: Optional, but I think being specific here is helpful 🙂
```suggestion
   Terms and conditions are **not** required to create a quotation template.
```

contain (e.g. :guilabel:`Template Header`, :guilabel:`Product`).

To edit the content, appearance, and overall design of the quotation template via the *Website*
application, click the :guilabel:`Edit` button in the upper-right corner.

.. image:: quote_template/design-template-edit-button.png
   :align: center
   :alt: Design template edit button in the upper-right corner of quotation template design.

When :guilabel:`Edit` is clicked, Odoo reveals a sidebar filled with a variety of design elements
and feature-rich building blocks. These building blocks can be dragged-and-dropped anywhere on the
quotation template design.

.. image:: quote_template/design-quotation-building-blocks.png
   :align: center
   :alt: Design quotation template building blocks sidebar in Odoo Website.

After a block has been dropped in the desired position, it can be customized and configured to fit
any unique need, look, or style.

.. tip::
   Quotation template design uses the same methodology and functionality with design building blocks
   as a typical web page design with Odoo *Website*. Be sure to check out the

>    :doc:`/applications/websites/website` documentation to learn more.

Reviewer 4: Update doc path to be relative
```suggestion
   :doc:`../../../websites/website` documentation to learn more.
```

When all blocks and customizations are complete, click the :guilabel:`Save` button to put those
configurations into place.

There is also a blue banner at the top of the quotation template design with a link to quickly
return :guilabel:`Back to edit mode`. When clicked, Odoo returns to the quotation template form in
the back-end of the *Sales* application.

Use quotation templates
=======================

When creating a quotation (:menuselection:`Sales app --> New`), choose a pre-configured template in
the :guilabel:`Quotation Template` field.

.. image:: quote_template/quotation-templates-field.png
   :align: center
   :alt: Quotation templates field on a standard quotation form in Odoo Sales.

To view what the customer will see, click the :guilabel:`Preview` button at the top of the page to
see how the quotation template appears on the front-end of the website through Odoo's customer
portal.

.. image:: quote_template/quotations-templates-preview.png
   :align: center
   :alt: Customer preview of a quotation template in Odoo Sales.

.. seealso::

>    - :doc:`/applications/sales/sales/send_quotations/get_signature_to_validate`

Reviewer 4: Update doc paths to be relative
```suggestion
   - :doc:`get_signature_to_validate`
   - :doc:`get_paid_to_validate`
```
